src/Main/App.js

```diff
@@ -282,6 +287,8 @@ class App extends Component {
       config: null,
       parser: null,
       progress: 0,
+      showSelectorPlayers: false,
```

Comment:
I think this state should be in `PlayerSelectorHeader`, that way all the toggling logic is managed by one component and it's reusable as a whole.